src/_base/harness/attributes/common.yml

```
@@ -1,53 +1,234 @@
+      branch: develop
+      cluster:
+        name: null
+      namespace: = @('workspace.name') ~ '-' ~ 'qa'
+      hostname: = @('pipeline.qa.namespace') ~ '.example.com'
 
   docker:
     registry:
-      url: ~
-      username: ~
-      password: ~
-    repository: ~
+      url: = get_docker_registry(@('docker.repository'))
+      username: = @('docker.username') # for backwards compatibility
+      password: = @('docker.password') # for backwards compatibility
+    repository: = @("workspace.name")
+    buildkit:
+      enabled: true
+    compose:
+      file_version: '3.7'
+      host_volume_options: "= ':' ~ (@('delegated-volumes') ? 'delegated' : 'cached')"
+    # If using gitops helm chart repositories, it's recommended to put this configuration
+    # in the values.yml in there now instead of project application repositories
+    image_pull_config: = @('docker.config')
+    image:
+      node: = 'node:' ~ @('node.version') ~ '-slim'
+      nginx: nginx:1.17-alpine
+    port_forward:
+      enabled: "= @('app.build') == 'dynamic'"
+
+
+  framework:
+    readme_blocks: []
+
+  frontend:
+    path: /app
+    watch: npm run watch
+    build:
+      when: -f "package.json"
+      steps:
+        - |
+          if [ "$APP_BUILD" == "static" ] && [ -f package-lock.json ]; then
+            run npm clean-install
+          else
+            run npm install
+          fi
+        - |
+          if [ "$(jq ".scripts.build != null" < package.json)" != "false" ]; then
+            run npm run build
+          fi
+        - |
+          if [ "$APP_BUILD" == "static" ]; then
+            run rm -rf node_modules/
+            run npm cache clean --force
+          fi
 
   git:
-    main_branch: develop
+    default_branch: develop
+
+  lighthouse:
+    target:
+      url: = 'https://' ~ @('hostname')
+    success-thresholds:
+      pwa:
+        enabled: false
+        score: 0
+      seo:
+        enabled: false
+        score: 0
+      best-practices:
+        enabled: false
+        score: 0
+      accessibility:
+        enabled: false
+        score: 0
+      performance:
+        enabled: false
+        score: 0
+
+  node:
+    # only set this attribute if you wish to override the supplied node version, by default
+    # the supplied version will be the current LTS.
+    version: lts
+
+  assets:
+    remote: ="s3://"~@("aws.bucket")~"/development"
+    local: tools/assets/development
+
+  domain: my127.site
+  hostname: = @('namespace') ~ '.' ~ @('domain')
+  hostname_aliases: []
 
   helm:
-    image_pull_config: = docker_config(@('docker.registry'))
     additional_schema_locations: https://inviqa.github.io/kubernetes-json-schema/schema
-
     feature:
-      sealed_secrets: true
+      # Note: be very careful considering disabling this, as in most cases
+      # it causes the secrets in it to be stored plaintext on filesystem
+      # or in helm chart repositories
+      # requires sealed-secrets k8s operator
+      sealed_secrets: false
```

```suggestion
      sealed_secrets: true
```
We meant to turn this on by default in php harness v1.0.0, but forgot.